content/en/blog/news/kitex_performance_testing/index.md

```diff
@@ -1,85 +1,128 @@
+Recently, ByteDance Service Framework Team officially announced the open source of CloudWeGo. It includes the Golang microservice RPC framework -- Kitex, which has been deeply used in Douyin and Toutiao.
 
-## The Charaterstics of Microservice Scenario  
+This article aims to share the scenarios and technical issues that developers need to know when stress testing Kitex.
+These guides will help users adjust and optimize Kitex to better match their business needs, and maximize Kitex's performance in real RPC scenarios.
+Users can also refer to the official stress test project -- [kitex-benchmark](https://github.com/cloudwego/kitex-benchmark) for more details.
 
-* **RPC Communication Model**  
-The communication between microservices is usually based on PingPong model. So, in addition to the conventional throughput performance index, developers also need to consider the average latency of each RPC.  
+## The Characteristics of Microservice Scenario
 
-* **Complex Call Chain**  
-A RPC request usually requires multiple microservices to work together. With downstream services having their own dependencies, it leads the whole call chain to become a complex network structure. In these kind of complex call chains, the latency fluctuation of one intermediate node may be transmitted to the whole and lead to increase of overall latency. When there are many nodes on the chain, even if the latency of each node is less likely to fluctuate, the overall timeout probability of the chain could be amplified. Therefore, the latency fluctuation of a single service, notably P99, is also a key indicator that has a significant impact on online services.  
+Kitex was born in ByteDance's large-scale microservices architecture practice. The scenario it is aimed at is naturally a microservices scenario.
+Therefore, the following will first introduce the characteristics of microservices, so that developers can understand Kitex's design thinking in depth.
 
-* **Size of Data Package**  
-Although the size of transmitted data packages depends on the actual business scenario, the internal statistics of ByteDance found that most online requests are small packages (<2KB). So we focused on optimizing the performance in the small data package scenarios while taking the large package scenarios into account.  
+* **RPC Communication Model**
+An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.
 
-## Stress Test for Microservice Scenarios  
+* **Complex Call Chain**
+The communication between microservices is usually based on PingPong model. So, in addition to the conventional throughput performance index, developers also need to consider the average latency of each RPC.
```

Comment:
An RPC call often requires multiple microservices to collaborate, and downstream services have their own dependencies,  so the entire call chain will be a complex network structure.